src/MaaWpfGui/Res/Localizations/ko-kr.xaml

```diff
@@ -688,12 +688,12 @@ Bilibili: 로그인 인터페이스에 표시되는 계정 이름(예: 장산)
     <system:String x:Key="QqChannel">QQ 채널</system:String>
     <system:String x:Key="Telegram">Telegram</system:String>
     <system:String x:Key="Help">자주 묻는 질문</system:String>
-    <system:String x:Key="Issue">이슈</system:String>
+    <system:String x:Key="Issue">이슈 제보</system:String>
     <!--  !About  -->
     <!--  AutoRecruitSettings  -->
     <system:String x:Key="AutoRefresh">★3 태그를 자동 새로고침하기</system:String>
-    <system:String x:Key="ForceRefresh">채용 허가 없이 계속 태그 새로고침 시도</system:String>
-    <system:String x:Key="AutoUseExpedited">즉시 완료 허가증을 자동 사용*</system:String>
+    <system:String x:Key="ForceRefresh">채용없이 태그 새로고침 반복 시도</system:String>
+    <system:String x:Key="AutoUseExpedited">즉시 완료 허가증을 자동 사용</system:String>
```

Comment:
The `*` means that the settings will not be saved after reboot, the same as UseStone on the homepage.

Reply:
Ah, I see. I have made the corrections and committed it again